ignite/handlers/checkpoint.py

```diff
@@ -287,7 +288,7 @@ def __init__(
         self.ext = "pt"
         self.global_step_transform = global_step_transform
         self.filename_pattern = filename_pattern
-        self._saved = []
+        self._saved: list = []
```

Comment:
We still provide support for py3.5 and this syntax is unfortunately unsupported. Can we avoid doing this ?

Reply:
We can transform it to this

```suggestion
        self._saved = []  # type: list
```